Win11ThemeTest/TextBoxTest.cs

```diff
@@ -0,0 +1,449 @@
+
+        #region UITests
+
+        [Test]
+        public void tbb1_clickOnTextbox()
+        {
+            Mouse.Click(textBox.GetClickablePoint());
+            Assert.That(textBox.IsEnabled, Is.True);
+        }
+
+        [Test]
+        public void tbb2_mouseHover()
+        {
+            Mouse.MoveTo(textBox.GetClickablePoint());
+            Assert.That(textBox.Text, Is.EqualTo("Hello World!Hello World!Hello World!Hello World!Hello World!Hell World!Hello World!Hello World!Hello World!"));
+        }
+
+        [Test]
+        public void tbb3_fontFamily()
+        {
+            using (var automation = new UIA3Automation())
+            {
+                string expected_FontFamily = "Segoe UI";
+                textBox.Enter("Hello World!");
+                var colorRange = textBox.Patterns.Text.Pattern;
+                var textFont = (string)colorRange.DocumentRange.GetAttributeValue(automation.TextAttributeLibrary.FontName);
+                Assert.That(textFont, Is.EqualTo(expected_FontFamily));
+
+            }
+        }
+
+        [Test]
+        public void tbb4_fontSize()
+        {
+            using (var automation = new UIA3Automation())
+            {
+                // string expected_FontFamily = "Segoe UI";
+                double expected_FontSize = 14;
+                textBox.Enter("Hello World!");
+                var colorRange = textBox.Patterns.Text.Pattern;
+                var textFontSize = colorRange.DocumentRange.GetAttributeValue(automation.TextAttributeLibrary.FontSize);
+                //var backColorInt = colorRange.DocumentRange.GetAttributeValue(TextAttributes.FontSize);
+                Console.WriteLine("textFontSize...........{0}", textFontSize);
+                //Console.WriteLine("backColorInt...........{0}", backColorInt);
+                Assert.That(textFontSize, Is.EqualTo(expected_FontSize));
+            }
+        }
+
+        [Test]
+        public void tbb5_textForegroundColor()
+        {
+            var automation = new UIA3Automation();
+            textBox.Enter("Hello World!");
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(500));
+            var expectedColor = System.Drawing.ColorTranslator.FromHtml("#E4000000");
+            var pattern = textBox.Patterns.Text.Pattern;
+            var foreColorInt = (int)pattern.DocumentRange.GetAttributeValue(TextAttributes.ForegroundColor);
+
+            var actualColor = Color.FromArgb(foreColorInt);
+            Console.WriteLine("actualColor........{0}", actualColor);
+            AssertColorEquality(actualColor, Color.FromArgb(0, expectedColor));
+        }
+
+        [Test]
+        public void tbb6_textBackgroundColor()
+        {
+            using (var automation = new UIA3Automation())
+            {
+                // textBox.Enter("Hello World!");
+                var pattern = textBox.Patterns.Text.Pattern;
+                var expectedColor = System.Drawing.ColorTranslator.FromHtml("#B3CB1B1B");
```

Comment:
Simplify member access and remove commented code.

Reply:
Changed and removed comments.